### ISSUE TITLE
Fix headings on hompage, footer, explore data, query data

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2732]
- [x] **Homepage:** About natural resources revenue data should be H2 and we need a separate H1
---
- [x]  **Footer**: Built in the Open needs to be H2
---
- [x]  **How Revenue Works**: Ownership, Leasing, Production, Revenue & Disbursements boxes should be H3
---
- [ ]  **Explore data**: Map section needs a heading, Card labels are not a heading but should be H3 (sections on cards are H4), Add Location and Explore ... visually are the same as explore data but they are not headings, second level headings on map cards are H6, but the second level headings on Compare cards are H4
---
- [x]  **Query Data:** Query Data and Download look the same - one is an H1 and the other is not

**Comment by Maroyafaied** [comment]
@cthomasONRR please also update the content audit spreadsheet.

**Comment by cthomasONRR** [comment]
For How Revenue Works, I don't see a location to change H2 to H3. I tried to add H3 here but the build fails
![image](https://github.com/DOI-ONRR/nrrd/assets/72526208/86d52fed-a777-430a-9893-99a2ba344ddf)
I see H2 for "How our data fits together", but not a location for headings within the cards.

**Comment by cthomasONRR** [comment]
I have heading updates to the homepage & footer in PR #2850.

I think the heading updates in How Revenue Works, Explore Data, and Query Data might need Jeff's help. 
- In How Revenue Works, the cards are formatted outside of regular html, maybe in css. I'm not sure how to add headings there.
- For Explore Data and Query Data, I'm not sure the correct file to update. Updates might need to be in a .js file that I'm not familiar with.

**Comment by Maroyafaied** [comment]
@cthomasONRR The headings for how revenue works should be H3 now. Here's a screenshot of where I made the code change. 
![image](https://github.com/DOI-ONRR/nrrd/assets/40175688/989bc166-05fc-48ee-ab35-2af6c096e518)

**Comment by Maroyafaied** [comment]
@cthomasONRR The "Download" in query tool should be H1 now. Here's a screenshot of where I added the code. 
![image](https://github.com/DOI-ONRR/nrrd/assets/40175688/6944b0e0-f59d-4f61-8ddb-095887892d83)

**Comment by Maroyafaied** [comment]
@cthomasONRR @eelzi-ONRR I want to make sure I'm understanding , should the card title be H3 in explore data? For example in the screenshot below, do we want "Native American" to be H3? and what about the "USA" underneath it? 
![image](https://github.com/DOI-ONRR/nrrd/assets/40175688/87d11aee-f393-4b25-8785-da01fdbe450d)

**Comment by cthomasONRR** [comment]
@eelzi-ONRR Can you help answer this?

**Comment by Maroyafaied** [comment]
@cthomas I add heading (h1) to add a location and explore more in explore data. It should work for all data types after it's done building. 
![image](https://github.com/DOI-ONRR/nrrd/assets/40175688/3941af37-d130-4dc6-ab06-18decde8b204)

![image](https://github.com/DOI-ONRR/nrrd/assets/40175688/5adf9f47-03a9-45ee-a5fc-a8b34faf8a00)

**Comment by eelzi-ONRR** [comment]
@Maroyafaied Yes, Native American is the H3. USA doesn't need to be included in that - it's just giving additional clarification on what the heading refers to.

**Comment by cthomasONRR** [comment]
@Maroyafaied The H1s look good: 
![image](https://github.com/DOI-ONRR/nrrd/assets/72526208/baee0439-33c9-4d1e-9c27-cd4eaa392fef)

**Comment by cthomasONRR** [comment]
I moved the Explore Data requests here into issue #2858
